packages/terra-overlay/src/Overlay.scss

```diff
@@ -50,10 +54,10 @@
   }
 
   .content {
-    color: rgba(28, 31, 33, 1);
+    color: var(--terra-overlay-content-color, #1c1f21);
     font-size: 1rem;
     max-height: 100%;
-    padding: 1rem;
+    padding: var(--terra-overlay-content-padding, 1rem);
```

Comment:
Can we break this out into 
* `--terra-overlay-content-padding-top`
* `--terra-overlay-content-padding-left`
* `--terra-overlay-content-padding-right`
* `--terra-overlay-content-padding-bottom`

to be consistent with [this](https://github.com/cerner/terra-core/blob/master/packages/terra-form-textarea/src/Textarea.scss#L19-L22).